src/app/shared/ui/modals/example dialog/example-dialog.component.html

```diff
@@ -0,0 +1,3 @@
+<!-- Komponent stworzony na potrzeby wyśietlenia dialogów oraz przekazania do nich danych, w wersji produkcyjnej do usuniecia -->
+      <button mat-raised-button (click)="openDialog()">Dialog z inputem</button>
+      <button mat-raised-button (click)="openDialog2()">Dialog z przyciskami</button>
```

Comment:
jeszcze jedna kwestia - jesli nazwa folderu sklada sie z wiecej niz 1 członu, to oddzielamjmy przez myslnik